### PR TITLE
Spin: Fixes for issues reported by API tests

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/855
[Problem] The API tests of Spin for wearable profile fail
[Solution]
     - fixed paths for resources incudet to API tests
     - fixed Spin widget issues which occurred after split widget to two profiles

This PR depends from https://github.com/Samsung/TAU/pull/820

### CODE REVIEW

File: src/js/profile/wearable/widget/wearable/NumberPicker.js [hunk 1]
```diff
@@ -46,7 +46,7 @@
 				selectors = ns.util.selectors,
 				prototype = new ns.widget.BaseWidget(),
 				Spin = ns.widget.wearable.Spin,
-				SPIN_CLASS = "ui-spin",
+				SPIN_CLASS = Spin.classes.widget,
```
Comment (HunseopJeong) [inline src/js/profile/wearable/widget/wearable/NumberPicker.js]: Is the '.widget' corect value in the 'Spin.classes'?

Comment (TomaszLukawskiSam) [reply src/js/profile/wearable/widget/wearable/NumberPicker.js]: You are right. Incorrect.

Comment (TomaszLukawskiSam) [reply src/js/profile/wearable/widget/wearable/NumberPicker.js]: Fixed.

**Review by HunseopJeong — APPROVED** [review]
Looks good to me!
I will merge this PR after #820.

**Comment by HunseopJeong** [comment]
@TomaszLukawskiSam,
Could you check the conflicts?

**Comment by TomaszLukawskiSam** [comment]
> @TomaszLukawskiSam,
> Could you check the conflicts?

Done. Tests passed.

**Pull request merged**